docs/pages/enroll-resources/application-access/introduction.mdx

```diff
@@ -39,6 +39,7 @@ CLI tools with the same RBAC system you use to protect your infrastructure.
 
 - [AWS Console and CLI Applications](./cloud-apis/aws-console.mdx): How to access
   AWS Management Console, AWS CLI, and AWS SDKs with Teleport.
+- [Google Cloud Web Console](../../admin-guides/access-controls/idps/saml-gcp-workforce-identity-federation.mdx): How to access Google Cloud web console with Teleport.
```

Comment:
```suggestion
- [Google Cloud Web Console](../../admin-guides/access-controls/idps/saml-gcp-workforce-identity-federation.mdx): How to access the Google Cloud web console with Teleport.
```
"Google Cloud web console" needs an article since it's not a proper name